data/json/speech.json

```diff
@@ -1757,6 +1757,90 @@
     "sound": "a soft robotic voice say, \"Here we go.  Just hold still.\"",
     "volume": 8
   },
+  {
+    "type": "speech",
+    "speaker": "mon_grocerybot_running_out_of_friendship",
+    "sound": "a voice say, \"I don't want to go but our time together is drawing to an end.\"",
```

Comment:
```suggestion
    "sound": "a voice says, \"I don't want to go but our time together is drawing to an end.\"",
```

Reply:
`You hear a voice say "...` Sounds better than `You hear a voice says "..`

Reply:
My mistake, it seems I had simply misremembered the context in which these would have been displayed.